content/v1.12/concepts/managed-resources.md

```diff
@@ -214,6 +214,14 @@ under `spec` should look like.
   deleted when this managed resource is deleted in Kubernetes API server.
   Possible values are `Delete` (the default) and `Orphan`.
 
+* `managementPolicy`: Enum to specify the level of control Crossplane has over
+  the external resource.
+  Possible values are `FullControl` (the default) and `ObserveOnly`.
+  {{<hint "important">}}
+  `managementPolicy` is an experimental feature, see the management policies
+  section below for further details.
```

Comment:
Don't we have a specific `alphaFeature` front-matter thing for this?

Reply:
Can you elaborate a bit, not sure I get it correctly?

Reply:
@negz we can't add the flag in this context since it applies to an entire page. That would mark "managed resources" as alpha.